internal/mods.json

```diff
@@ -221,6 +221,57 @@
           "asset_index": 15
         }
       ]
+    },
+    {
+        "name": "Unique Rockets",
+        "blurb": "Unique rockets for Soldiers Viewmodels",
+        "icon": "https://fastdl.creators.tf/launcher/assets/unique_rockets_icon_1.png",
+        "titleimage": "",
+        "backgroundimage": "https://fastdl.creators.tf/launcher/assets/unique_rockets_bg.jpg",
```

Comment:
Always load images from local side to avoid possible errors. If necessary we can just add a fallback method to load images from the internet when the URL changes.